src/modules/sdk/objections/service.ts

```diff
@@ -35,12 +36,12 @@ const OBJECTIONS_API_SINGLE_ATTACHMENT_URL =
  * @returns {string} the id of the newly created objection
  * @throws {ApiError}
  */
-export const createNewObjection = async (companyNumber: string, token: string): Promise<string> => {
+export const createNewObjection = async (companyNumber: string, token: string, create: ObjectionCreate): Promise<string> => {
```

Comment:
can we rename the create params, to something like createWithData

Reply:
in other functions as well

Reply:
Renamed.